Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM debian:stable-slim
+MAINTAINER Vincent Giersch <vincent@flat.io>
+
+RUN mkdir /faust
+WORKDIR /faust
+COPY . /faust
```

Comment:
I was wondering if it's intentional that the faust source is copied before the apt installs, which means that the base software is re-installed whenever the faust code changes. This makes it extremely slow to test faust code changes locally, but I can see advantages: the latest debian software versions are always fetched when the source code has been updated. I'm just getting into Docker, so I was wondering if this ordering is considered best practice.